### ISSUE TITLE
Build is very slow

### ISSUE THREAD

**Issue by squelix** [sveltejs/kit#5784]
### Describe the bug

Since i update `@sveltejs/kit` from `1.0.0-next.397` to `1.0.0-next.401` the build with `@sveltejs/adapter-netlify` is very very slow. You can see the time on the logs.

### Reproduction

https://github.com/squelix/svelte-portfolio on main, you can run `npm run build` and see that it's very slow.

### Logs

```shell
9:09:26 AM: $ npm run build
9:09:26 AM: npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
9:09:26 AM: npm WARN config location in the cache, and they are managed by
9:09:26 AM: npm WARN config     [`cacache`](http://npm.im/cacache).
9:09:26 AM: > svelte-portfolio@0.0.1 build
9:09:26 AM: > vite build
9:09:26 AM: vite v3.0.2 building for production...
9:09:26 AM: transforming...
9:09:28 AM: 7:09:28 AM [vite-plugin-svelte] /opt/build/repo/src/lib/commons/Nav-item.svelte:36:0 Unused CSS selector ".item__link--selected"
9:09:28 AM: 34:   width: 100%;
9:09:28 AM: 35: }
9:09:28 AM: 36: .item__link--selected {
9:09:28 AM:     ^
9:09:28 AM: 37:   position: relative;
9:09:28 AM: 38:   font-style: unset;
9:09:28 AM: 7:09:28 AM [vite-plugin-svelte] /opt/build/repo/src/lib/commons/Nav-item.svelte:41:0 Unused CSS selector ".item__link--selected::after"
9:09:28 AM: 39:   text-decoration: none;
9:09:28 AM: 40: }
9:09:28 AM: 41: .item__link--selected::after {
9:09:28 AM:     ^
9:09:28 AM: 42:   content: "";
9:09:28 AM: 43:   transition: width 0.3s ease-in-out;
9:09:29 AM: ✓ 277 modules transformed.
9:09:29 AM: rendering chunks...
9:23:03 AM: .svelte-kit/output/client/manifest.json                                          3.30 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/pages/__error.svelte-a714d657.js        1.72 KiB / gzip: 0.93 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/pages/_lang_/index.svelte-8c677c45.js   4.32 KiB / gzip: 1.65 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/routing-72051bf7.js              1.89 KiB / gzip: 0.68 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/start-8930e9ee.js                       23.38 KiB / gzip: 8.69 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/common-759cf503.js               0.10 KiB / gzip: 0.11 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/preload-helper-14ff6353.js       10.22 KiB / gzip: 4.13 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/common-beaf611a.js               0.09 KiB / gzip: 0.10 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/error-672f97cb.js                0.15 KiB / gzip: 0.14 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/stores-1df1a1ae.js               13.00 KiB / gzip: 4.67 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/error-8cc41f94.js                0.14 KiB / gzip: 0.13 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/home-a74962e2.js                 0.34 KiB / gzip: 0.25 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/chunks/home-ba797ec6.js                 0.32 KiB / gzip: 0.24 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/assets/index-b7ea55d8.css               0.94 KiB / gzip: 0.34 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/assets/__layout-e3a7c122.css            7.86 KiB / gzip: 2.57 KiB
9:23:03 AM: .svelte-kit/output/client/_app/immutable/pages/__layout.svelte-a673b7aa.js       183.60 KiB / gzip: 54.94 KiB
9:23:03 AM: vite v3.0.2 building SSR bundle for production...
9:23:03 AM: transforming...
9:23:04 AM: 7:23:04 AM [vite-plugin-svelte] /opt/build/repo/src/lib/commons/Nav-item.svelte:36:0 Unused CSS selector ".item__link--selected"
9:23:04 AM: 34:   width: 100%;
9:23:04 AM: 35: }
9:23:04 AM: 36: .item__link--selected {
9:23:04 AM:     ^
9:23:04 AM: 37:   position: relative;
9:23:04 AM: 38:   font-style: unset;
9:23:04 AM: 7:23:04 AM [vite-plugin-svelte] /opt/build/repo/src/lib/commons/Nav-item.svelte:41:0 Unused CSS selector ".item__link--selected::after"
9:23:04 AM: 39:   text-decoration: none;
9:23:04 AM: 40: }
9:23:04 AM: 41: .item__link--selected::after {
9:23:04 AM:     ^
9:23:04 AM: 42:   content: "";
9:23:04 AM: 43:   transition: width 0.3s ease-in-out;
9:23:04 AM: ✓ 50 modules transformed.
9:23:04 AM: rendering chunks...
9:23:04 AM: .svelte-kit/output/server/manifest.json                               3.18 KiB
9:23:04 AM: .svelte-kit/output/server/index.js                                    79.71 KiB
9:23:04 AM: .svelte-kit/output/server/entries/endpoints/index.ts.js               0.31 KiB
9:23:04 AM: .svelte-kit/output/server/entries/pages/__layout.svelte.js            16.84 KiB
9:23:04 AM: .svelte-kit/output/server/entries/pages/__error.svelte.js             1.49 KiB
9:23:04 AM: .svelte-kit/output/server/entries/pages/_lang_/index.svelte.js        3.88 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/index-cefb4a83.js     6.91 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/routing-a299db70.js   5.74 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/index-46a00aae.js     1.49 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/stores-eaa7d12c.js    0.63 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/hooks-0be3bcf3.js     0.68 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/common-759cf503.js    0.13 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/common-beaf611a.js    0.12 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/error-672f97cb.js     0.19 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/error-8cc41f94.js     0.18 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/home-a74962e2.js      0.44 KiB
9:23:04 AM: .svelte-kit/output/server/_app/immutable/chunks/home-ba797ec6.js      0.42 KiB
9:23:04 AM: Run npm run preview to preview your production build locally.
9:23:04 AM: 
9:23:04 AM: > Using @sveltejs/adapter-netlify
9:23:04 AM:   ✔ done
```

### System Info

```shell
System:
    OS: macOS 12.5
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.41 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 1.22.19 - ~/.yvm/shim/yarn
    npm: 8.15.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
  Browsers:
    Brave Browser: 101.1.38.109
    Chrome: 103.0.5060.134
    Edge: 103.0.1264.77
    Firefox: 94.0.1
    Safari: 15.6
    Safari Technology Preview: 16.0
  npmPackages:
    @sveltejs/adapter-netlify: 1.0.0-next.71 => 1.0.0-next.71 
    @sveltejs/kit: 1.0.0-next.401 => 1.0.0-next.401 
    svelte: ^3.49.0 => 3.49.0 
    vite: ^3.0.4 => 3.0.4 
```

### Severity

blocking an upgrade

**Comment by dominikg** [comment]
see #5786 
and check if removing https://github.com/squelix/svelte-portfolio/blob/main/src/app.d.ts#L1  helps

**Comment by squelix** [comment]
I did remove `<reference types="@sveltejs/kit" />` but the build is still slow :/

**Comment by benmccann** [comment]
Closing as a duplicate of https://github.com/sveltejs/kit/issues/5786. I confirmed it's the same issue

**Comment by squelix** [comment]
Thanks! This is fixed with 1.0.0-next.402
